public/js/qr/export.js

```
@@ -0,0 +1,125 @@
+            downloadOptions.image = await this.createBrandImage(this.currentBrandName);
+        }
+
+        const downloadQR = new QRCodeStyling(downloadOptions);
+
+        const tempContainer = document.createElement('div');
+        tempContainer.style.position = 'absolute';
+        tempContainer.style.left = '-9999px';
+        document.body.appendChild(tempContainer);
+
+        downloadQR.append(tempContainer);
+        await new Promise(resolve => setTimeout(resolve, 100));
```

ESLint

[error] 'setTimeout' is not defined.

(no-undef)